docs/source/support/frequently-asked-questions.rst

```diff
@@ -578,6 +578,44 @@ Here we provide a bit more context on how the ``run:`` commands are used. The ex
 
 After this step, the reusable workflow then runs the ``pytest`` command. To see the full reusable workflow file, please visit https://github.com/scikit-package/release-scripts/blob/main/.github/workflows/_tests-on-pr.yml.
 
+.. _faq-private-repo-no-codecov:
+
+I have a private repo and don't have a Codecov paid plan.  Can I modify the CI workflows for this situation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, the workflows in ``tests-on-pr.yml`` and ``matrix-and-codecov-on-merge-to-main.yml`` will have a section that uploads the code coverage to Codecov, and CI will fail if unsuccessful. If you have a private repository and Codecov cannot be run, CI will always fail, and this is undesirable.
```

Comment:
"....unsuccessful.  Codecov is available for free on public repositories but requires a paid plan to run on private repositories.  In this latter case CI will always fail because Codecov fails.  Scikit-package has alternative CI workflows for this situation."